.github/workflows/sanitize.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   sanitized-build:
-    name: Compile with multiple sanitizers
+    name: "/"
```

Comment:
The build names for sanitizer builds look like they have an extra `/` in them. Is that coming from here and is it intentional/expected?

Reply:
Yes.  I tried using `""`, but that is ignored and the job id (`sanitized-build`) is used instead. 

I can use any other separator if you think it would work better.  I did not try `" "`, let me see how that works.

Reply:
Looks nicer I think, thanks for making me think.